source/beamline/VESPERS/VESPERSEndstation.h

```diff
@@ -111,7 +114,7 @@ public slots:
 	/// Sets the current control that should be focused on.  It will find the correct control based on the name.  Control is set to 0 if invalid name is given.
 	void setCurrent(QString name);
 	/// Toggles the current state of the laser power.
-	void toggleLaserPower() { toggleControl(laserPower_); emit laserPoweredChanged(); }
+	void toggleLaserPower() { laserPower_->move(1); emit laserPoweredChanged(); }
```

Comment:
Just to make sure, setting the toggle PVs to 1 initiates a change in state from on to off or vice verse (it isn't a 1 to on, 0 to off deal)?